### ISSUE TITLE
Integration with Kubernetes

### ISSUE THREAD

**Issue by yanns** [hjr3/alacrity#49]
Kubernetes sends events when a new service is started and when it is not available anymore.
We could integrate alacrity to synchronize with the state of the cluster in Kubernetes.

**Comment by hjr3** [comment]
Great idea. I will do some reading on how this works. This probably means we need to abstract the adding and removal of nodes and make the REST API, Kubernetes, etc all plugins.

**Comment by yanns** [comment]
Yes or we have an additional component listening to K8S events, and using the REST API to synchronize.
